repositories/noteRepository.js

```diff
@@ -0,0 +1,39 @@
+const Note = require('../models/Note');
+
```

Comment:
I noticed that the folder is named "repositories." Is this naming pattern from a course or a specific guideline you followed? From my point of view, naming it "services" might be more meaningful, as it reflects the purpose here. Wdyt?

Reply:
I chose "repositories" because they directly handle data interactions, aligning with repository design patterns focused on CRUD operations. Adding a "service" layer could be the next step for managing more complex business logic from the controller.